include/extractor/extraction_turn.hpp

```diff
@@ -12,15 +12,20 @@ namespace extractor {
 
 struct ExtractionTurnLeg {
   ExtractionTurnLeg(bool is_restricted, TravelMode mode, bool is_motorway,
-                    bool is_link, int number_of_lanes, int highway_turn_classification, int access_turn_classification, int speed)
+                    bool is_link, int number_of_lanes,
+                    int highway_turn_classification,
+                    int access_turn_classification, int speed, bool is_incoming,
```

Comment:
i think you also need incoming/outgoing status for other modes. e.g for bicycle routing you also need information about car traffic

Reply:
Thanks! I'll target that in another issue. I'd prefer getting the changes ready and see how these changes make a difference in better ETA & route planning & extraction memory usage peak. But your comment is well noted and will not be forgotten! :)